### ISSUE TITLE
Blue Tinted Overlay When Using Hardware Keys to Navigate Chapter

### ISSUE THREAD

**Issue by Hero-Over** [tachiyomiorg/tachiyomi#5555]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.11.1)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.11.1-3583

### Android version

Android 11

### Device

Multiple Devices

### Steps to reproduce

1. Open Chapter
2. Use any hardware keys to navigate chapter, such as; Arrow Keys, Pg Up/Down

### Expected behavior

Unchanged manga image
https://imgur.com/DqmykV9

### Actual behavior

Blue Tinted Overlay on Image 
https://imgur.com/gJM0QfZ

### Other details

This did not happen in older builds

**Comment by kiriya-aoi** [comment]
Update to the latest Preview build (r3598) and try again to verify the issue persists.

**Comment by Hero-Over** [comment]
> Update to the latest Preview build (r3598) and try again to verify the issue persists.
Thank you for the response,

Just gave it a try and the issue still persists. This bug is also present on every recent fork of Tachiyomi.

I also seemed to have narrowed it down as I just did a little debugging and it seems as if the "tapping navigation" implementation uses an overlay that divides the portions and acts as an invisible but tangible interface element. So when using anything other than volume keys (I remapped a bit in the Generic.kl to see if I made the L/R arrows act as volume keys, then the issue would no longer persist and this is what motivated me to check the logs) you will end up "highlighting that input box that stretches over the entirety of the screen. The interface already registered touch input by location, so I don't see why the added overlay was necessary (could be wrong here) 

Now this is just my sob story. I personally loathe the dumb overlays that pop up incessantly and while I would rather do a small swipe, I don't know why the tap regions must have an aggressive input box instead of allocating any touch made within a certain percentage of the X resolution (By the way you developers are heroes). Man, I live my life as a front end web programmer and couldn't imagine embarking on such an ambitious project for free, except back when I was 19 and doing AOSP Mods and writing Kernels to make the shitty old Android phones decent. Even then I was paid quite well in donations. So THANK YOU ALL so very much for making my favorite hobby so simple and easy

Here are my HW input logs:
2021-07-19 10:26:49.855 4039-4400/? D/InputDispatcher: Inject key (12174): action=0, f=0x0, d=-1, time=1766154000000
2021-07-19 10:26:49.855 4039-4400/? D/WindowManager: interceptKeyTq s_result=1
2021-07-19 10:26:49.858 4039-4198/? I/InputDispatcher: Delivering key to (29428): DP action: 0x0 (0), f=0x0, m=0x200000, d=0, '2385797'
2021-07-19 10:26:49.859 12759-12759/? I/HBD: a skip onKeyDown as physical keyboard not connected 
2021-07-19 10:26:49.860 12759-12759/? I/HBD: HoneyBoardService onKeyDown consumed:false 
2021-07-19 10:26:49.861 29428-29428/? I/ViewRootImpl@ccfe0ed[ReaderActivity]: ViewPostIme key 0

and here are the logs after recovering the screen tint by touching the center of the screen after the hw keys triggered the overlay:
2021-07-19 10:30:55.781 29428-29428/? D/InsetsSourceConsumer: setRequestedVisible: visible=true, type=0, host=eu.kanade.tachiyomi.sy/eu.kanade.tachiyomi.ui.reader.ReaderActivity, from=android.view.InsetsSourceConsumer.show:229 android.view.InsetsController.showDirectly:1437 android.view.InsetsController.controlAnimationUnchecked:1110 android.view.InsetsController.applyAnimation:1417 android.view.InsetsController.show:962 android.view.InsetsController.show:918 android.view.PendingInsetsController.show:51 androidx.core.view.WindowInsetsControllerCompat$Impl30.show:1 eu.kanade.tachiyomi.ui.reader.ReaderActivity.setMenuVisibility:3 eu.kanade.tachiyomi.ui.reader.ReaderActivity.setMenuVisibility$default:1 
2021-07-19 10:30:55.782 4039-5107/? D/InsetsSourceProvider: updateVisibility: serverVisible=true, clientVisible=true, source=InsetsSource: {mType=ITYPE_NAVIGATION_BAR, mFrame=[0,1830][1200,1920], mVisible=true}, controlTarget=Window{2385797 u0 eu.kanade.tachiyomi.sy/eu.kanade.tachiyomi.ui.reader.ReaderActivity}, from=com.android.server.wm.InsetsSourceProvider.setClientVisible:378 com.android.server.wm.InsetsSourceProvider.onInsetsModified:363 com.android.server.wm.InsetsStateController.onInsetsModified:416 com.android.server.wm.InsetsPolicy.onInsetsModified:320 com.android.server.wm.Session.insetsModified:512 android.view.IWindowSession$Stub.onTransact:1398 com.android.server.wm.Session.onTransact:145 android.os.Binder.execTransactInternal:1195 android.os.Binder.execTransact:1159 <bottom of call stack> 
2021-07-19 10:30:55.791 4892-5735/? I/TouchInteractionService: onSystemUiStateChanged, stateFlags : 0
2021-07-19 10:30:55.792 4892-4892/? I/TouchInteractionService: onSystemUiFlagsChanged, userUnlocked
2021-07-19 10:30:55.792 4402-4402/? V/Navbar.Store: handleEvent [EventType] ON_NAVBAR_WINDOW_STATE_SHOWING [Module] NavigationBarFragment
2021-07-19 10:30:55.800 4039-5107/? D/InsetsSourceProvider: updateVisibility: serverVisible=true, clientVisible=true, source=InsetsSource: {mType=ITYPE_STATUS_BAR, mFrame=[0,0][1200,45], mVisible=true}, controlTarget=Window{2385797 u0 eu.kanade.tachiyomi.sy/eu.kanade.tachiyomi.ui.reader.ReaderActivity}, from=com.android.server.wm.InsetsSourceProvider.setClientVisible:378 com.android.server.wm.InsetsSourceProvider.onInsetsModified:363 com.android.server.wm.InsetsStateController.onInsetsModified:416 com.android.server.wm.InsetsPolicy.onInsetsModified:320 com.android.server.wm.Session.insetsModified:512 android.view.IWindowSession$Stub.onTransact:1398 com.android.server.wm.Session.onTransact:145 android.os.Binder.execTransactInternal:1195 android.os.Binder.execTransact:1159 <bottom of call stack> 
2021-07-19 10:30:55.807 4039-21217/? I/SystemUiVisibilityPolicyController: handleMessage: entry what = 1
2021-07-19 10:30:55.813 7178-7178/? I/TrayStateController: systemUiVisibilityChanged : 2 0
2021-07-19 10:30:55.932 4039-4183/? V/WindowManager: Relayout Window{2385797 u0 eu.kanade.tachiyomi.sy/eu.kanade.tachiyomi.ui.reader.ReaderActivity}: viewVisibility=0 req=1200x1920
2021-07-19 10:30:55.939 4039-4183/? V/WindowManager: Relayout hash=2385797, pid=29428: mAttrs={(0,0)(fillxfill) sim={adjust=pan} layoutInDisplayCutoutMode=shortEdges ty=BASE_APPLICATION wanim=0x1030304
      fl=81810180
      pfl=1a020000
      vsysui=2010
      apr=LIGHT_STATUS_BARS LIGHT_NAVIGATION_BARS
      bhv=SHOW_TRANSIENT_BARS_BY_SWIPE
      fitSides= naviIconColor=0}
2021-07-19 10:30:55.940 4039-4183/? I/SurfaceControl: nativeRelease nativeObject s[488684563936]
2021-07-19 10:30:55.940 4039-4183/? I/SurfaceControl: nativeRelease nativeObject e[488684563936]
2021-07-19 10:30:55.941 29428-29428/? I/SurfaceControl: nativeRelease nativeObject s[485530337120]
2021-07-19 10:30:55.941 29428-29428/? I/SurfaceControl: nativeRelease nativeObject e[485530337120]
2021-07-19 10:30:55.945 29428-29428/? I/SurfaceControl: assignNativeObject: nativeObject = 0 Surface(name=null)/@0xaffbb2c / android.view.SurfaceControl.readFromParcel:1117 android.view.IWindowSession$Stub$Proxy.relayout:1820 android.view.ViewRootImpl.relayoutWindow:9005 android.view.ViewRootImpl.performTraversals:3360 android.view.ViewRootImpl.doTraversal:2618 android.view.ViewRootImpl$TraversalRunnable.run:9971 android.view.Choreographer$CallbackRecord.run:1010 android.view.Choreographer.doCallbacks:809 android.view.Choreographer.doFrame:744 android.view.Choreographer$FrameDisplayEventReceiver.run:995 
2021-07-19 10:30:55.947 29428-29428/? I/SurfaceControl: nativeRelease nativeObject s[485524661472]
2021-07-19 10:30:55.948 29428-29428/? I/SurfaceControl: nativeRelease nativeObject e[485524661472]

I could be wrong about the culprit, as I haven't written for Android since the entire source code was basically just Java and front end relying on a near stock Linux Kernel
